cdap-master-spi/pom.xml

```diff
@@ -32,10 +32,19 @@
   <packaging>jar</packaging>
 
   <dependencies>
+    <dependency>
+      <groupId>co.cask.cdap</groupId>
+      <artifactId>cdap-proto</artifactId>
```

Comment:
This could complicate the classloading of master-spi implementation, as cdap-proto has dependency on guava. Can we not depends on it?

Reply:
cdap-proto only has a test dependency on guava. Otherwise it only depends on cdap-api and cdap-runtime-spi

Reply:
ugh, actually the dependency on cdap-api actually causes issues, because it depends on gson.